<launch>
  <arg name="google_cloud_credentials_json" default="$(optenv GOOGLE_APPLICATION_CREDENTIALS)" doc="Read credentials JSON from this value when use_yaml is false." />
  <arg name="project_id" default="eternal-byte-236613" />
  <arg name="run_app_manager" default="true" doc="Run app_manager or not"/>

  <include file="$(find dialogflow_task_executive)/launch/dialogflow_task_executive.launch" >
    <arg name="credential" value="$(arg google_cloud_credentials_json)" />
    <arg name="project_id" value="$(arg project_id)" />
    <arg name="run_app_manager" value="$(arg run_app_manager)" />
    <arg name="enable_hotword" default="false" />
  </include>
  <node pkg="dialogflow_task_executive" type="client.l"
        name="client" output="screen" />
</launch>
